src/guacscanner/guacscanner.py

```diff
@@ -682,24 +683,24 @@ def main() -> None:
     # Parse command line arguments
     args = docopt.docopt(__doc__, version=__version__)
     # Validate and convert arguments as needed
-    schema = Schema(
+    mySchema = schema.Schema(
```

Comment:
Nit-picky but Python conventions prefers snake case variable names per [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names)

```suggestion
    my_schema = schema.Schema(
```